src/Sql/dbo/Tables/UserSignatureKeyPair.sql

```diff
@@ -0,0 +1,11 @@
+﻿CREATE TABLE [dbo].[UserSignatureKeyPair] (
+    [Id]                        UNIQUEIDENTIFIER NOT NULL,
+    [UserId]                    UNIQUEIDENTIFIER NOT NULL,
+    [SignatureKeyPairAlgorithm] TINYINT NOT NULL,
+    [SigningKey]                VARCHAR(MAX) NOT NULL,
+    [VerifyingKey]              VARCHAR(MAX) NOT NULL,
+    [CreationDate]              DATETIME2 (7) NOT NULL,
+    [RevisionDate]              DATETIME2 (7) NOT NULL,
+    CONSTRAINT [PK_UserSignatureKeyPair] PRIMARY KEY CLUSTERED ([Id] ASC),
+    CONSTRAINT [FK_UserSignatureKeyPair_User] FOREIGN KEY ([UserId]) REFERENCES [dbo].[User] ([Id]),
+);
```

Comment:
```
GO
CREATE NONCLUSTERED INDEX [IX_UserSignatureKeyPair_UserId]
    ON [dbo].[UserSignatureKeyPair]([UserId] ASC);

```
💭 

If we will be searching by UserID it might make sense to have an index for that column. 

Entity Framework Core will be automatically generating these indexes for us on the other DBs.

I'll leave the final decision to the DB team on this.

Reply:
Added the index, great callout!